soap-attachments/1.4/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta SOAP with Attachments Specification 1.4"
+date: 2019-07-23T14:30:32+0200
+summary: "Initial release of Jakarta SOAP with Attachments"
+---
+Jakarta SOAP with Attachments defines an API enabling developers to produce
+and consume messages conforming to the SOAP 1.1, SOAP 1.2, and SOAP Attachments Feature.
+
+* [Jakarta SOAP with Attachments 1.4 Specification Document](./jakarta.xml.soap-spec-1.4.pdf) (PDF)
```

Comment:
Similar comment as the other PRs...  The name of the spec should be soap-spec-1.4.pdf to be consistent with our [naming doc.](https://github.com/jakartaee/specification-committee/blob/master/names.adoc)

Reply:
done